src/main/java/com/vadim/tkach/lesson18/Vehicle.java

```diff
@@ -0,0 +1,36 @@
+package com.vadim.tkach.lesson18;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.time.LocalDate;
+@Getter
```

Comment:
apply code auto-formatting: https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html